index.js

```diff
@@ -1,28 +1,24 @@
 'use strict'
 
-const path = require('path')
-
-const defaultLocal = {
-  dir: '$HOME/.persist/'
-}
-
-const defaultPersisters = [ 's3', 'local' ]
+const defaultPersisters = ['s3', 'local']
 
 class Persister {
+
   /**
    * Creates persisters for saving and loading
    *
    * @param  {Object} opts Description of persisters
    * @return {Persist}     Persister
    */
+
   constructor(services) {
     this.persisters = []
 
-    services.forEach(service => {
-      if(defaultPersisters.indexOf(service.type) > -1) {
-        let persisterModule = require(`./lib/persisters/${service.type}`)
+    services.forEach((service) => {
+      if (defaultPersisters.indexOf(service.type) > -1) {
+        const PersisterModule = require(`./lib/persisters/${service.type}`)
 
-        this.persisters.push(persisterModule(service))
+        this.persisters.push(new PersisterModule(service))
       } else {
         this.persisters.push(service.implementation)
```

Comment:
this means that custom services don't follow the same pattern as the built-in ones? is there a reason for that? the client could just pass an object that implements `save` and `load`. depends how you want it to work I guess